packages/core/source/elements/RemoteElement.ts

```diff
@@ -643,7 +643,7 @@ export abstract class RemoteElement<
       attribute === 'slot' &&
       (this.constructor as typeof RemoteElement).slottable
```

Comment:
Do we need this guard in the property setter? If not, would we get into a situation where the component implementation reflects the attribute even though it's not supported?

Reply:
I don't think we want to prevent a `slot` attribute/ property from getting written in the same way they would be for the DOM. This additional, `RemoteElement`-specific `slottable` property is just for deciding whether that attribute should be synchronized with the host. I kind of wish I had not made this optional at all, since I can't imagine a reason why you would not want to send `slot` to the host, especially since the host needs to guard against unexpected attributes anyways.